CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 4.3.1-0.9.8_3-0.13.8_2-r0 (UNRELEASED)
+
+* ruTorrent v4.3.1
+  * Fix RSS downloader button and RSS feeds disabled status
+  * Add PHP extension warnings for dom and libxml
+  * Increase Material Design theme font visibility
+
```

Comment:
Please don't make changes in the changelog, this is done only when releasing but you can add it to your PR description